tests/author.test.js

```diff
@@ -0,0 +1,158 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const { Author } = require('../src/models');
+const app = require('../src/app');
+
+describe('/authors', () => {
+    before(async () => Author.sequelize.sync());
+
+    beforeEach(async () => {
+        await Author.destroy({ where: {} });
+    });
+
+    describe('with no records in the database', () => {
+        describe('POST /authors', () => {
+            it('creates a new author in the database', async () => {
+                const response = await request(app).post('/authors').send({
+                    author: 'Octavia E. Butler',
+                });
+
+                expect(response.status).to.equal(201);
+                expect(response.body.author).to.equal('Octavia E. Butler');
+            });
+
+            it('responds with an error message if the author string is empty', async() => {
+                const noAuthor = await request(app).post('/authors').send({
+                    author: ''
+                });
+                expect(noAuthor.status).to.equal(400);
+                expect(noAuthor.body.error).to.equal('Please enter the author of the book');
+            })
+
+            it('responds with an error message if a non unique author is entered', async() => {
+                const firstAuthor = await request(app).post('/authors').send({
+                    author: 'Brandon Sanderson'
+                });
+                const secondAuthor = await request(app).post('/authors').send({
+                    author: 'Brandon Sanderson'
+                });
+
+                expect(firstAuthor.status).to.equal(201);
```

Comment:
Minor point but the convention with arrangement of test assertions is to group them all together, leaving a blank line before the first one. Would be nice to put that in place across all the tests